python/assistant-stream/src/assistant_stream/create_run.py

```
@@ -10,21 +10,23 @@
 
 class RunController:
     def __init__(self, queue):
-        self.queue = queue
-        self.loop = asyncio.get_event_loop()
-        self.stream_tasks = []
+        self._queue = queue
+        self._loop = asyncio.get_event_loop()
+        self._dispose_callbacks = []
+        self._stream_tasks = []
 
     def append_text(self, text_delta: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `append_text`

Add return type annotation: `None`

(ANN201)